master/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml

```diff
@@ -50,6 +50,7 @@ rules:
       - watch
   - apiGroups: ["crd.projectcalico.org"]
     resources:
+      - globalfelixconfigs
```

Comment:
@song-jiang could you also add in `globalbgpconfigs` - some deployments will need those as well.

Reply:
Sure.